### DISCUSSION TITLE
Is there any way to support import playlists and rating star from iTunes

### DISCUSSION THREAD

**Discussion by panshuo** [jeffvli/feishin#708] [Ideas]
same as above

**Comment by kgarner7** [comment]
Given that Feishin just gets metadata (aside from lyrics) from your server, no. It would be much better to import that to your server (Navidrome/Jellyfin)

**Comment by nova77** [comment]
If you're running navidrome, I wrote [a simple library](https://github.com/nova77/musicdb_parser) which parse iTunes/Apple Music libraries to extract all sorts of metadata. [One of the examples](https://github.com/nova77/musicdb_parser/blob/master/navidrome/update_annotations.py) updates a navidrome library with it.

**Comment by panshuo** [comment]
Thank you so much for your work.I'll try it this weekend.😃

Norman ***@***.***> 于2026年2月7日周六 13:26写道：
